### PR TITLE
feat(auto-complete): adds append prop

### PR DESCRIPTION
**Pull request by cinyearchan**
[adds append prop for auto-complete](https://github.com/tusen-ai/naive-ui/issues/5591)

### CODE REVIEW

**Comment by OrbisK** [comment]
image
you example does not open a select

**Comment by cinyearchan** [comment]
> image you example does not open a select

use it in conjunction with get-show, in the example, the dropdown menu will appear when typing the @ symbol, append the selected option to the input box instead of directly overwriting it, achieve the same effect as google search box

**Comment by OrbisK** [comment]
> > image you example does not open a select
> 
> use it in conjunction with get-show, in the example, the dropdown menu will appear when typing the @ symbol, append the selected option to the input box instead of directly overwriting it, achieve the same effect as google search box

ahhh nvm. missed the getShow 😄

**Review by OrbisK — APPROVED** [review]

**Review by songjianet — APPROVED** [review]

File: src/auto-complete/demos/enUS/index.demo-entry.md [hunk 1]
```diff
@@ -21,6 +22,7 @@ status.vue
 
 | Name | Type | Default | Description | Version |
 | --- | --- | --- | --- | --- |
+| append | `boolean` | `false` | Whether to append content to input. |  |
```
Comment (07akioni) [inline src/auto-complete/demos/enUS/index.demo-entry.md:25]:
```suggestion
| append | `boolean` | `false` | Whether to append content to input. | NEXT_VERSION |
```

File: src/auto-complete/src/AutoComplete.tsx [hunk 2]
```diff
@@ -62,6 +62,10 @@ export const autoCompleteProps = {
   ...(useTheme.props as ThemeProps<AutoCompleteTheme>),
   to: useAdjustedTo.propTo,
   menuProps: Object as PropType<HTMLAttributes>,
+  append: {
+    type: Boolean,
+    default: false
+  },
```
Comment (07akioni) [inline src/auto-complete/src/AutoComplete.tsx:68]:
```suggestion
  append: Boolean,
```

Comment (07akioni) [reply src/auto-complete/src/AutoComplete.tsx:68]: Write it directly if default value is false.

File: src/auto-complete/demos/zhCN/index.demo-entry.md [hunk 3]
```diff
@@ -21,6 +22,7 @@ status.vue
 
 | 名称 | 类型 | 默认值 | 说明 | 版本 |
 | --- | --- | --- | --- | --- |
+| append | `boolean` | `false` | 选中后是否追加 |  |
```
Comment (07akioni) [inline src/auto-complete/demos/zhCN/index.demo-entry.md:25]:
```suggestion
| append | `boolean` | `false` | 选中后是否追加 | NEXT_VERSION |
```

**Pull request merged**